kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -131,6 +133,15 @@
       currentUserEdit: null,
     }),
     computed: {
+      LEARNER() {
```

Comment:
Read only UserKinds here. I thought this'd be a nice way to cut down on the repetitious `UserKinds.*`, but let me know if you see a problem with this. Did the same thing in the modals.